Formula/o/openai-whisper.rb

```diff
@@ -79,24 +81,22 @@ class OpenaiWhisper < Formula
   end
 
   resource "urllib3" do
-    url "https://files.pythonhosted.org/packages/43/6d/fa469ae21497ddc8bc93e5877702dca7cb8f911e337aca7452b5724f1bb6/urllib3-2.2.2.tar.gz"
-    sha256 "dd505485549a7a552833da5e6063639d0d177c04f23bc3864e41e5dc5f612168"
+    url "https://files.pythonhosted.org/packages/ed/63/22ba4ebfe7430b76388e7cd448d5478814d3032121827c12a2cc287e2260/urllib3-2.2.3.tar.gz"
+    sha256 "e7d814a81dad81e6caf2ec9fdedb284ecc9c73076b62654547cc64ccdcae26e9"
   end
 
   def install
-    python3 = "python3.12"
-    venv = virtualenv_create(libexec, python3)
+    ENV["LLVM_CONFIG"] = Formula["llvm@16"].opt_bin/"llvm-config"
+    venv = virtualenv_install_with_resources without: "numba"
 
     # We depend on pytorch, but that's a separate formula, so install a `.pth` file to link them.
-    # This needs to happen _before_ we try to install torchvision.
-    site_packages = Language::Python.site_packages(python3)
-    pytorch = Formula["pytorch"].opt_libexec
-    (venv.site_packages/"homebrew-pytorch.pth").write pytorch/site_packages
+    # NOTE: This is an exception to our usual policy as building `pytorch` is complicated
+    site_packages = venv.site_packages.relative_path_from(venv.root)
+    pth_contents = "import site; site.addsitedir('#{Formula["pytorch"].opt_libexec/site_packages}')\n"
```

Comment:
```suggestion
    pth_contents = "#{Formula["pytorch"].opt_libexec/site_packages}\n"
```
I _think_ this is equivalent?

Reply:
In current state, they will behave similarly, but `site.addsitedir` is recursive while the latter is not.

Only becomes an issue when there are recursive usage of pth files, but I think it is safer to just keep the `site.addsitedir`, which also aligns with how we autogenerate the pth

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/language/python.rb#L200